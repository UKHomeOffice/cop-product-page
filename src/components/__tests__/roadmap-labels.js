import React from "react";
import renderer from "react-test-renderer";
import RoadMapLabel from "../roadmap-labels";

describe("roadmapLabel", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<RoadMapLabel text={"roadmap item"} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct RoadMapLabel 'text' prop passed in", () => {
    const tree = renderer.create(<RoadMapLabel text={"roadmap Label"} />);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty("text", "roadmap Label");
  });
});
