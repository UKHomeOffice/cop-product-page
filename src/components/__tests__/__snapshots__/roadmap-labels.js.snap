// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`roadmapLabel renders correctly 1`] = `
<div
  className="grid-row"
>
  <div
    className="govuk-grid-column-one-third"
  >
    <div
      className="govuk-body"
      style={
        Object {
          "fontSize": "14px",
        }
      }
    >
      roadmap item
    </div>
  </div>
</div>
`;
